src/marqo/s2_inference/s2_inference.py

```diff
@@ -100,18 +100,28 @@ def _validate_model_properties(model_name: str, model_properties: dict) -> dict:
     if model_properties is not None:
         """checks model dict to see if all required keys are present
         """
-        required_keys = ["name", "dimensions"]
-        for key in required_keys:
-            if key not in model_properties:
-                raise InvalidModelPropertiesError(f"model_properties has missing key '{key}'. ")
-
-        """updates model dict with default values if optional keys are missing
-        """
-        optional_keys_values = [("type", "sbert"), ("tokens", get_default_seq_length())]
-        for key, value in optional_keys_values:
-            if key not in model_properties:
-                model_properties[key] = value
-
+        if model_properties.get("type", None) in (None, "sbert"):
+            required_keys = ["name", "dimensions"]
+            for key in required_keys:
+                if key not in model_properties:
+                    raise InvalidModelPropertiesError(f"model_properties has missing key '{key}'."
+                                                      f"please update your model properties with required key `{key}`"
+                                                      f"check `https://docs.marqo.ai/0.0.12/Models-Reference/dense_retrieval/` for more info.")
```

Comment:
Can we be as specific and helpful as possible? 

For example, linking the specific section of the documentation: 
https://docs.marqo.ai/0.0.12/Models-Reference/dense_retrieval/#generic-models

and what sort of information they can find there `check htts://docs.marqo... for examples of model_properties`